examples/advanced/tailwind-preset/tailwind.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
Tailwind supports ESM which is the more modern format, so let's use that everywhere instead of CJS